### DISCUSSION TITLE
Adbyby Plus+ NOT RUNNING

### DISCUSSION THREAD

**Discussion by ivanors** [helmiau/HelmiWrt-OS#45] [General]
bang helmi,ada bug di bagian Adbyby Plus+,sudah di enable tapi tidak jalan (Adbyby Plus+ NOT RUNNING)

Device Raspberry PI 4 B

![bug adbyby plus](https://user-images.githubusercontent.com/81950468/135885693-b5ee5444-41dd-4b53-aba4-3807c079dd25.jpg)

**Comment by helmiau** [comment]
sepertinya ini bug dari adbyby plusnya om

**Reply by ivanors** [reply]
sepertinya dari fw nya om,di device lain berjalan normal

**Reply by rootjack404** [reply]
Provide more detail such as system log, application log, or anything that might be useful other than just screenshot of it not working which doesn't provide insight to the problem at all.
